tests/test_combobox.py

```diff
@@ -19,3 +19,7 @@ def test_set_selected(self):
         value = 0
         self.combobox.setSelected(value)
         self.assertEqual(self.combobox.selected(), value)
+
+    def test_can_pass_items_to_constructor(self):
+        """Tests user can pass items to the constructor of the combobox."""
+        combobox = Combobox(['Blue', 'Yellow', 'Red'])
```

Comment:
I wondered that too. At least we know the Combobox constructor does not raise a TypeError like `TypeError: __init__() takes 1 positional argument but 2 were given` if we pass a list as a parameter.
Although I agree this may not be very useful without any assert... :)